src/engine.c

```diff
@@ -130,7 +130,6 @@ ENGINE_readFile(ENGINE* engine, char* path, size_t* lengthPtr) {
       return file;
     } else {
       ENGINE_printLog(engine, "Error reading %s: %s\n", pathBuf, mtar_strerror(err));
```

Comment:
We probably want to use the original fall-back message, rather than the bundle-specific one.